lib/cequel/schema/table.rb

```diff
@@ -13,20 +13,20 @@ class Table
       # @return [Symbol] the name of the table
       attr_reader :name
 
-      # @return [Array<Column>] all columns defined on the table
-      attr_reader :columns
+      # # @return [Array<Column>] all columns defined on the table
+      # attr_reader :columns
 
       # @return [Array<PartitionKey>] partition key columns defined on the
       #   table
-      attr_reader :partition_key_columns
+      # attr_reader :partition_key_columns
 
       # @return [Array<ClusteringColumn>] clustering columns defined on the
       #   table
-      attr_reader :clustering_columns
+      # attr_reader :clustering_columns
 
       # @return [Array<DataColumn,CollectionColumn>] data columns and
       #   collection columns defined on the table
-      attr_reader :data_columns
+      # attr_reader :data_columns
```

Comment:
let's delete these if they aren't needed anymore.

Reply:
ok.. i'll push this change.